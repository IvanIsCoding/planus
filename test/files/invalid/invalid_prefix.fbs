table Foo {
  x0: [bool] = +[];
  x1: bool = +true;
  x2: uint8 = +null;
  x3: string = +"hello";
  x4: float = +Hello;
}