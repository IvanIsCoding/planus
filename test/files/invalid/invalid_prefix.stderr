[0m[1m[38;5;9merror[0m[1m: Cannot use prefix sign on lists[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_prefix.fbs:2:16
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x0: [bool] = [0m[31m+[][0m;
  [0m[34mâ”‚[0m                [0m[31m^^^[0m

[0m[1m[38;5;9merror[0m[1m: Cannot use prefix sign on booleans[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_prefix.fbs:3:14
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m   x1: bool = [0m[31m+true[0m;
  [0m[34mâ”‚[0m              [0m[31m^^^^^[0m

[0m[1m[38;5;9merror[0m[1m: Cannot use prefix sign on null[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_prefix.fbs:4:15
  [0m[34mâ”‚[0m
[0m[34m4[0m [0m[34mâ”‚[0m   x2: uint8 = [0m[31m+null[0m;
  [0m[34mâ”‚[0m               [0m[31m^^^^^[0m

[0m[1m[38;5;9merror[0m[1m: Cannot use prefix sign on strings[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_prefix.fbs:5:16
  [0m[34mâ”‚[0m
[0m[34m5[0m [0m[34mâ”‚[0m   x3: string = [0m[31m+"hello"[0m;
  [0m[34mâ”‚[0m                [0m[31m^^^^^^^^[0m

[0m[1m[38;5;9merror[0m[1m: Cannot use prefix sign on constants[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_prefix.fbs:6:15
  [0m[34mâ”‚[0m
[0m[34m6[0m [0m[34mâ”‚[0m   x4: float = [0m[31m+Hello[0m;
  [0m[34mâ”‚[0m               [0m[31m^^^^^^[0m

Error: Bailing because of errors
